### PR TITLE
@ashfurrow => Show Pages

### PR DESCRIPTION
**Pull request by orta**
Adds support for showing installation images on a show page, then you can scroll down into the artworks.

screen shot 2015-10-13 at 17 49 15

screen shot 2015-10-13 at 17 49 11

### CODE REVIEW

File: Emergence/Contexts/App Setup/AppViewController.swift [hunk 1]
```diff
@@ -13,31 +13,38 @@ class AppViewController: UINavigationController {
         return AppContext(network:network, auth:auth)
     }()
 
-    // TODO: What about if app is launching from scratch,
-    //       and they are in the auth slideshow?
-
     func openShowWithID(showID: String?) {
+        func topVCIsAuth() -> Bool {
+            guard let topVC = self.topViewController else { return true }
```
Comment (ashfurrow) [inline Emergence/Contexts/App Setup/AppViewController.swift:18]: This returns `true` if the top VC exists, is that right?

**Comment by ashfurrow** [comment]
Looks great @orta! :cake:

File: Emergence/Contexts/App Setup/AppViewController.swift (hunk 1 above)
Comment (orta) [inline Emergence/Contexts/App Setup/AppViewController.swift:18]: hrm, no, that feels off to me

Comment (orta) [inline Emergence/Contexts/App Setup/AppViewController.swift:18]: oh yeah, it's fine :+1:

**Pull request merged**